authors/apps/authentication/tests/test_login.py

```diff
@@ -0,0 +1,62 @@
+from rest_framework.views import status
+
+from authors.apps.authentication.tests.base_test import BaseTest
+
+
+class TestLogin(BaseTest):
+    '''
+    Tests for the login functionality of the app
+    '''
+
+    def test_successful_user_login(self):
+        '''
+        test for a successful user login
+        '''
+        self.signup_a_user(self.user_data)
+        response = self.login_a_user(self.user_login_data)
+        self.assertEqual(response.data["email"], "jam@gmail.com")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
Check for the status code first in all the tests since if the status fails there's no need to keep testing.

Reply:
Working on it